doc/DeveloperGuide.md

```diff
@@ -88,6 +88,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename an existing tag
+
+**Preconditions**
+
+1. AddressBook is populated with at least one person
+2. There exists at least one person with a tag
+
+**Guarantees**
+
+1. All instances of the specified tag will be renamed
```

Comment:
Is it really guarantees? :P 
What if it goes into the 5a. Extension, then its not renamed, 

The correct statement is `All instances of the specified tag will be renamed if successful`

P.S. Having `Guarantees` is not necessary so feel free to avoid it when unsure. Anyways good attempt :)